src/main/java/bio/terra/service/profile/ProfileDao.java

```diff
@@ -41,6 +41,8 @@
         + " FROM billing_profile"
         + " WHERE id in (:idlist)";
 
+    private static final String sqlListOld = "SELECT " + sqlSelectList
```

Comment:
I haven't read the the update design doc yet, but using names like "old" and "new" don't work well once you have more than one version. Better to have specific numbers/names for versions, and use those so it's clear what version the code refers to.